### ISSUE TITLE
Crash rendering node with style 'line-height:normal'

### ISSUE THREAD

**Issue by jh3141** [archriss/react-native-render-html#164]
### Is this a bug report or a feature request?

Bug report

### Have you read [the guidelines](https://github.com/archriss/react-native-render-html/blob/master/CONTRIBUTING.md) regarding bug report?

Yes

### Have you read [the documentation](https://github.com/archriss/react-native-render-html/blob/master/README.md) in its entirety?

Yes

### Have you made sure that your issue hasn't already been reported/solved?

Yes.  My issue may be related to https://github.com/archriss/react-native-render-html/issues/108 but that issue was closed for lack of information.

### Is the bug specific to iOS or Android? Or can it be reproduced on both platforms?

Only tested on Android; reproducible on all Android SDK versions tested (21, 23, 24).

### Is the bug reproductible in a production environment (not a debug one)?

Yes (causes application crash)

### Have you been able to reproduce the bug in [the provided example](https://github.com/archriss/react-native-render-html/tree/master/Demo)?

Yes

### Environment

Environment:
    React: 16.3.0-alpha.1
    React native: 0.54.2
    react-native-render-html: 3.10.0

  Target Platform:
    Android (5.0, 6.0, 7.0)

### Steps to Reproduce

(Write your steps here:)

1. Create a component with the following code in its render method:

```
    const html="<p style=\"line-height:normal\">Rob Rawlings is working hard to break the cycle of drug addiction in his family.</p>";
    return <HTML html={html} />;
```
2. Run an application that displays the component
3.

### Expected Behavior

HTML is rendered normally

### Actual Behavior

Red box error showing:

Error while upading property 'lineHeight' in shadow node of type: RCTVirtualText

java.lang.String cannot be cast to java.lang.Double

### Reproducible Demo

https://snack.expo.io/ryqCfGbb7

(causes expo to crash when run)

**Comment by Exilz** [comment]
Yeah this all boils down to #154 and the numerous other issues that have been opened because of the lack of filtering on those kind of styles.

Once I have some times to dig into this, I'll make sure to filter out once and for all styles that aren't compatible with react native. The best thing to do would be some kind of dictionary that would look like a typescript interface that checks all styles you're trying to render before everything falls apart.

This could be quite a lot of work so if someone feels like giving it a shot... it would be my pleasure to review some code.

**Comment by Exilz** [comment]
Fixed in `v4.1.1`
